recipes/newspaper3k/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "newspaper3k" %}
+{% set version = "0.2.5" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "abe2ff199e5a41cc9d690fd5ade7cf605dc542f98b1071b26e91ef136a50042d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
```

Comment:
Consider adding `noarch: python`

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
This is not applicable with the `skip` statement. I removed this by updating the PR myself. Sorry for the mistake.

Reply:
Mh, I think noarch is possible with skipping build in the build/run environment, or is this only true for cb3?

Reply:
I'm not sure what `cb3` is, but very recently I built a package `noarch: python` that was py3 only and found that I could install it on py2. This prompted a change in the docs:

https://github.com/conda-forge/conda-forge.github.io/pull/483

stating that `skip` selectors were not allowed with `noarch`.

Reply:
cb = conda-build3

> stating that skip selectors were not allowed with noarch.

This is true, but you can pin your run-time python to `>=3.3`